Makefile

```diff
@@ -0,0 +1,31 @@
+OS := $(shell uname)
+
+build: clean
+	go build
+
+test:
+	go test -v
+
+clean:
+	go clean
+	rm -f align
+	rm -f bin/*
+
+install: clean
+ifeq ($(OS),Darwin)
+	./build.sh darwin
+	cp -f bin/align-darwin /usr/local/bin/align
+endif 
+ifeq ($(OS),Linux)
+	./build.sh linux
+	cp -f bin/align-linux /usr/local/bin/align
+endif
+ifeq ($(OS),FreeBSD)
+	./build.sh freebsd
+	cp -f bin/align-freebsd /usr/local/bin/align
+endif
+uninstall: 
+	rm -f /usr/local/bin/align*
```

Comment:
In hindsight, this might not be good... If the user has anything else that might happen to match that patter, we're gonna nuke it too.